.github/workflows/github-ci.yml

```diff
@@ -33,6 +33,9 @@ jobs:
       - name: Build with Maven
         run: mvn -B clean verify
 
+      - name: Test with Maven
+        run: mvn test
```

Comment:
This is not required, because the previous `mvn verify` also includes running the tests.